internal/provider/common_cert.go

```diff
@@ -386,5 +386,12 @@ func createSubjectDistinguishedNames(ctx context.Context, subject certificateSub
 		result.SerialNumber = subject.SerialNumber.ValueString()
 	}
 
+	if !subject.EmailAddress.IsNull() && !subject.EmailAddress.IsUnknown() {
+		result.ExtraNames = append(result.ExtraNames, pkix.AttributeTypeAndValue{
+			Type:  []int{1, 2, 840, 113549, 1, 9, 1},
```

Comment:
I know the underlying type is an alias for `[]int`, but would still prefer to use the aliased type here from `encoding/asn1`:
```go
Type: asn1.ObjectIdentifier{1, 2, 840, 113549, 1, 9, 1},
```